conf/conf.json.sample

```diff
@@ -7,8 +7,7 @@
         },
         "binlog": {
                 "binlog-dir": "{RADON-BINLOG-DIR}",
-                "enable-binlog": true,
-                "enable-relay": true
+                "enable-binlog": true
```

Comment:
The binlog is used for relay by others, if we move enable-relay, the enable-binlog is useless.